package.json

```diff
@@ -31,14 +31,13 @@
   },
   "homepage": "https://github.com/cypress-io/cypress-chrome-recorder#readme",
   "dependencies": {
-    "@puppeteer/replay": "^2.6.1",
+    "@puppeteer/replay": "^2.10.2",
     "chalk": "^5.0.1",
     "execa": "^6.1.0",
     "globby": "^13.1.1",
     "inquirer": "^8.2.1",
     "is-git-clean": "^1.1.0",
-    "meow": "^10.1.2",
-    "puppeteer": "^19.2.2"
```

Comment:
Why was puppeteer removed to fix indentation?

Reply:
Apologies, I removed it because it is not used and it's not related to the indentation fix.

Reply:
@OrKoN It looks like puppeteer is a[ peer dependency](https://github.com/puppeteer/replay/blob/main/package.json#L89) of `@puppeteer/replay` so we need to keep this.

Reply:
It’s an optional peer dependency and it should only be needed for create Puppeteer-based runner extensions. As far as I see this module offers only a stringify extension. Is there a runner extension that I have overlooked?